benchmark/benchmark-atomic.cpp

```diff
@@ -0,0 +1,270 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// This file is intended to provide an interface for comparing the performance
+// of RAJA's atomic implementations with Desul's atomic implementations.  In order
+// to accomplish this without requiring two separate build system configurations
+// this file directly includes "desul/atomics.hpp" and directly calls desul namespace
+// atomics.  This is different from how a typical RAJA user would call a desul atomic.
+
+#include "RAJA/RAJA.hpp"
+#include "RAJA/util/for_each.hpp"
+#include "RAJA/util/Timer.hpp"
+#if defined (RAJA_ENABLE_OPENMP)
+#include "RAJA/policy/openmp/atomic.hpp"
+#include "RAJA/policy/openmp/policy.hpp"
```

Comment:
When we include "RAJA/RAJA.hpp" doesn't it include the headers listed below?

Reply:
it should

Reply:
RAJA.hpp pulls in all the RAJA headers and each one is guarded, so it won't be included more than once in a translation unit. It's not clear to me that the other include statements are pulling in anything.